frontend/app/routes/$lang+/_protected+/alerts+/index.tsx

```diff
@@ -0,0 +1,24 @@
+import type { LoaderFunctionArgs } from '@remix-run/node';
+import { redirect } from '@remix-run/node';
+
+import { getSubscriptionService } from '~/services/subscription-service.server';
+import { featureEnabled } from '~/utils/env.server';
+import { UserinfoToken } from '~/utils/raoidc-utils.server';
+import { getPathById } from '~/utils/route-utils';
+
+export async function loader({ context: { session }, params, request }: LoaderFunctionArgs) {
+  featureEnabled('email-alerts');
+
+  const userInfoToken: UserinfoToken = session.get('userInfoToken');
+  const alertSubscription = await getSubscriptionService().getSubscription(userInfoToken.sin ?? '');
+
+  if (!alertSubscription) {
+    return redirect(getPathById('$lang+/_protected+/alerts+/subscribe+/index', params));
+  } else if (alertSubscription.subscribed === true) {
+    return redirect(getPathById('$lang+/_protected+/alerts+/manage+/edit', params));
+  } else {
+    return redirect(getPathById('$lang+/_protected+/alerts+/subscribe+/confirm', params));
+  }
+}
+
+export default function AlertsIndex() {}
```

Comment:
Since this route only does redirecting, there's no need to export a component.

Reply:
removed